assets/css/main.css

```diff
@@ -325,10 +334,38 @@ html, body {
     border-color: #bce8f1;
 }
 
+@media (min-width: 1200px) and (max-width: 1800px) {
+    .frontpage .card {
+        width: 23%;
+    }
+}
+
+@media (min-width: 960px) and (max-width: 1200px) {
```

Comment:
I wonder if it would make sense if these matched the Bootstrap ones which affect our other layouts - https://getbootstrap.com/docs/3.4/css/#grid-media-queries

Reply:
I thought about this too and actually tried those. I also tried to find information of if there are any newer, "standard" breakpoints, but they seemed to vary among different sources. The breakpoints I used looked better for these course-cards, so that is the reason I went for those. 

I guess this is about whether we want to standardize the breakpoints throughout, with a bit worse appearance on the front page or customize these here.